matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -160,26 +160,32 @@ async fn message_loop<M: Messenger>(
 
         select! {
             _  = &mut timeout => {
-                requests_sender.unbounded_send(PeerRequest::KeepAlive).expect("send failed");
-                // UNWRAP: we will only ever get here if there already was a timeout
-                let interval = keep_alive_interval.unwrap();
-                timeout = Either::Left(Delay::new(interval)).fuse();
+                // We don't care if a KeepAlive fails to send
+                if let Err(e) = requests_sender.unbounded_send(PeerRequest::KeepAlive) {
+                    warn!("failed to send KeepAlive: {e:?}");
+                }
+                if let Some(interval) = keep_alive_interval {
+                    timeout = Either::Left(Delay::new(interval)).fuse();
+                }
             }
 
             message = events_receiver.next().fuse() => {
                 if let Some(event) = message {
                     debug!("{event:?}");
                     match event {
                         PeerEvent::IdAssigned(peer_uuid) => {
-                            id_tx.try_send(peer_uuid.to_owned()).unwrap();
+                            id_tx.try_send(peer_uuid.to_owned()).map_err(error::MessageSendError::PeerId)?;
```

Comment:
If signature was `Result<(), MessageSendError>` we could coerce here

Also, can be just `MessageSendError::PeerId`